src/main/java/net/syscon/elite/api/resource/MovementResource.java

```diff
@@ -64,7 +76,6 @@ MovementCount getRollcountMovements(
     @GetMapping("/{agencyId}/enroute")
     @ApiOperation(value = "Enroute prisoner movement details.", notes = "Enroute to reception", nickname = "getEnrouteOffenderMovements")
     @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "OK", response = MovementCount.class, responseContainer = "List"),
```

Comment:
quite a few of these - where the swagger docs are plain wrong and wrong type is returned